osu.Game/Overlays/MusicController.cs

```diff
@@ -281,9 +292,9 @@ private void play(BeatmapInfo info, bool isNext)
 
         private void updateDisplay(WorkingBeatmap beatmap, bool? isNext)
         {
-            BeatmapMetadata metadata = beatmap.Beatmap.Metadata;
-            title.Text = metadata.TitleUnicode ?? metadata.Title;
-            artist.Text = metadata.ArtistUnicode ?? metadata.Artist;
+            BeatmapMetadata metadata = beatmap.Beatmap.BeatmapInfo.Metadata;
+            title.Text = config.GetUnicodeString(metadata.Title, metadata.TitleUnicode);

```

Comment:
not sure, let's leave it there for now. probably want a centralised Unicode class or similar.